CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ Before anything else, thank you. Thank you for taking some of your precious time
 
 ## Tests
 
-See the `package.json` scripts section for how to run each category of tests.
+Now type `yarn run` or see the `package.json` scripts section for how to run each category of tests.
```

Comment:
I'd prefer if we didn't recommend the use of `yarn` and use `npm` instead. Because it's the standard.

Reply:
I think the `yarn` usage is correct currently because the project is developed with yarn (the `yarn.lock` file, CI builds). That said I would be fine with changing to npm to be more standard.

Reply:
`yarn` is a pretty popular alternative to `npm`. I would go for documenting usage of both, or none. Eventually only `npm`. But definitely not `yarn` only.

Here, listing available scripts with npm is possible with `npm run` as well as with yarn

Reply:
Right now, the project uses yarn. If we want to switch to npm that's fine, but it's a separate issue to improving the current docs.